### PR TITLE
Spelling: "I.E", "which decay"

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: po4a [hunk 1]
```diff
@@ -392,7 +392,7 @@ Do not change the POT and PO files, only the translation may be updated.
 
 Keeps the existing translation files even if the translation doesn't meet the threshold specified by --keep.
 This option does not create new translation files with few content,
```
Comment (comradekingu) [inline po4a]: For some odd reason there are two whitespaces here.

Comment (mquinson) [reply po4a]: The only reason is my mistake, I guess. I'll fix it.

Comment (mquinson) [reply po4a]: I fail to see the two whitespaces, sorry. Feel free to submit another PR.

Comment (comradekingu) [reply po4a]: You see them in the diff, but not anywhere else. Possibly zero-width spaces?

**Pull request merged**